dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/LogcatLogHandler.kt

```diff
@@ -49,5 +99,12 @@ internal class LogcatLogHandler(
             Log.ERROR to "E",
             Log.ASSERT to "A"
         )
+
+        private val ANONYMOUS_CLASS =
+            Pattern.compile("(\\$\\d+)+$")
+        private const val MAX_TAG_LENGTH = 23
+
+        private const val DEFAULT_LOGGER_CALLER_STACK_INDEX = 7
+        private const val LOGGER_CLASS_NAME = "com.datadog.android.log.Logger"
```

Comment:
Instead of hardcoding this you could have used `Logger::class.java.canonicalName`, that way you'd be prepared for naming changes.
But nevermind you can merge as is

Reply:
Wanted to do that...actually I forgot to add it, tks for the heads up